models/offer_type.py

```diff
@@ -234,7 +234,7 @@ def as_dict(self):
         'conditionalFields': []
     }
     INSTRUMENT = {
-        'proLabel': "Vente d’instruments de musique",
+        'proLabel': "Vente et location d'instruments de musique",
```

Comment:
```suggestion
        'proLabel': "Vente et location d’instruments de musique",
```
Et pour tous les autres.